### ISSUE TITLE
[Legitimate Site Unblocked]

### ISSUE THREAD

**Issue by Seyedehfatemehmir** [MetaMask/eth-phishing-detect#10047]
**Domain:**

**Details:** *Some details about the domain (optional)*

- [ ] I have checked to make sure that [there is not a duplicate issue](https://github.com/MetaMask/eth-phishing-detect/issues)

**Comment by Seyedehfatemehmir** [comment]
How can I have access to it?

**Comment by deshvin** [comment]
No domain has been listed. Please submit a new issue with the domain included if you're still having problems
